spec

```diff
@@ -559,9 +559,8 @@ endseg
 beginseg
     name "ovl_select"
     include "build/src/overlays/gamestates/ovl_select/z_select.o"
-    include "build/data/overlays/gamestates/z_select.data.o"
-    include "build/data/overlays/gamestates/z_select.rodata.o"
     include "build/data/overlays/gamestates/z_select.reloc.o"
+    //include "build/src/overlays/gamestates/ovl_select/ovl_select_reloc.o"
```

Comment:
what's this?

Reply:
its the auto generated reloc. it cant be used right now because of the non matching